integration-tests/tests/billing-projects.js

```diff
@@ -0,0 +1,92 @@
+const _ = require('lodash/fp')
+const { click, clickable, dismissNotifications, findText, noSpinnersAfter, select, signIntoTerra } = require('../utils/integration-utils')
+const { withUserToken } = require('../utils/terra-sa-utils')
+
+
+class BillingProjectsPage {
```

Comment:
I imagine my choice of using a class and calling methods on it in the test will be controversial. I am trying to move my tests toward the page pattern, which I have found to be a very nice way to write browser-based tests in the past.

https://martinfowler.com/bliki/PageObject.html

Reply:
asking out of curiosity, why are classes undesirable for the UI code?

Reply:
One thing to clarify: I wrote the previous comment after too quick of a look at the PR, with the misunderstanding that `BillingProjectsPage` is a (React) class component, which it isn't. It's just a class. If your question is, why use function components over class components, the answer may be something like:
- class components come pre-loaded with stuff that you may not need
- function components are more light-weight and help you minimize state (which is a major pillar of UI development and then some)
- can be hard to reuse stateful logic between components
- I read things here and there that there may be reduced support for class components

Even though `BillingProjectsPage` is not a component, I'd be curious if we could do away with it by having mocked values stored somewhere and pure functions retrieve them.